ush/setup.py

```diff
@@ -219,9 +219,13 @@ def add_jobname(tasks):
 
     # Load one more if running Coupled AQM
     if cfg_d['cpl_aqm_parm']['CPL_AQM']:
-        cfg_aqm = get_yaml_config("config_defaults_aqm.yaml")
+        cfg_aqm = get_yaml_config(Path(ushdir, "config_defaults_aqm.yaml"))
         update_dict(cfg_aqm, cfg_d)
 
+    # Load CCPP suite-specific settings
+    ccpp_suite = cfg_d['workflow']['CCPP_PHYS_SUITE']
+    ccpp_cfg = get_yaml_config(Path(ushdir, "ccpp_suites_defaults.yaml")).get(ccpp_suite, {})
```

Comment:
Yes, I think it could be optional. For example, only 4 of the 8 options are defined in that file. At this point, those are the only ones that need special treatment. It is possible that we will never add options for all the supported suites since one is "default" and may not need additional changes applied anywhere.